migrations/2021-08-23-161500_aggregated_ft_nft_tables/up.sql

```diff
@@ -0,0 +1,62 @@
+CREATE TABLE aggregated__fungible_tokens
+(
+    id                                bigserial                NOT NULL,
+    included_in_transaction_hash      text                     NOT NULL,
+    included_in_transaction_timestamp numeric(20, 0)           NOT NULL,
+    transaction_status                execution_outcome_status NOT NULL,
+    issued_contract_id                text                     NOT NULL,
```

Comment:
It might make sense, but from a first glance I cannot say what it is, so it might be worth considering a better name (I will need to read FT NEP before I can provide any suggestions)